test/cdk-lib/bedrock/agent.test.ts

```diff
@@ -83,35 +86,67 @@ beforeAll(() => {
     },
   };
 
+  actionGroupFunction = new lambda.Function(stack, 'ActionGroupFunction', {
+    code: lambda.Code.fromAsset('test/path'),
+    runtime: lambda.Runtime.NODEJS_LATEST,
+    handler: 'index.handler',
+  });
+
+  NagSuppressions.addResourceSuppressions(
+    actionGroupFunction,
+    [
+      {
+        id: 'AwsSolutions-IAM4',
+        reason: 'ActionGroup Lambda uses the AWSLambdaBasicExecutionRole AWS Managed Policy.',
+      },
+    ],
+    true,
+  );
+
+  const actiongroup = new bedrock.AgentActionGroup(stack, 'actionGroups', {
+    actionGroupName: 'test-action-group',
+    description: 'Use these functions to get information about the books in the Project Gutenburg library.',
+    actionGroupState: 'ENABLED',
+    actionGroupExecutor: actionGroupFunction,
+    apiSchema: bedrock.ApiSchema.fromInline('mock schema'),
+  });
+
   agent = new bedrock.Agent(stack, 'Agent', {
     foundationModel: bedrock.BedrockFoundationModel.ANTHROPIC_CLAUDE_V2_1,
     instruction: 'You provide support for developers working with CDK constructs.',
-    knowledgeBases: [kb],
```

Comment:
this has been re-added, we can uncomment the line

Reply:
done - a5b222881d2860c503ed38fcd68ea6619e07542a